### PR TITLE
New notebook for data in this repo.

### PR DESCRIPTION
**Pull request by scottdangelo**
The data is now in this repository. Change notebook and example
to pull from here.

### CODE REVIEW

File: notebooks/OpenScale.ipynb [hunk 1]
```diff
@@ -1437,20 +1437,13 @@
...
-   "outputs": [],
-   "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.5 with Spark 2.1",
-   "language": "python",
-   "name": "python3-spark21"
+   "display_name": "Python 3.6 with Spark",
```
Comment (markstur) [inline notebooks/OpenScale.ipynb:1444]: your readme is specifically asking for 3.5, but looks like you bumped the notebook to 3.6

File: notebooks/OpenScale.ipynb [hunk 2]
```diff
@@ -136,15 +136,15 @@
    "outputs": [],
    "source": [
     "WML_CREDENTIALS = {\n",
-    "  \"apikey\": \"kjfkdjfkdjfkdjkdjfkdjfkdfj\",\n",
-    "  \"iam_apikey_description\": \"Auto generated apikey during resource-key operation for Instance - crn:v1:bluemix:public:pm-20:us-south:a/fkjkfdkfjkdjkdjfkdjfkdfjdkfabb::\",\n",
-    "  \"iam_apikey_name\": \"auto-generated-apikey-kdjfkjdkdjf99938493893e3\",\n",
+    "  \"apikey\": \"***,\n",
+    "  \"iam_apikey_description\": \"Auto generated apikey during resource-key operation for Instance - crn:v1:bluemix:public:pm-20:us-south:a/47b84451ab70b94737518f7640a9ee42:4a7f62cc-f564-4144-8af4-b9f53533d2cf::\",\n",
```
Comment (markstur) [inline notebooks/OpenScale.ipynb]: not sure what the guid is.  OK or wanna go back to fkjkfdkf or ****?

File: notebooks/OpenScale.ipynb [hunk 3]
```diff
@@ -35,7 +35,7 @@
     "!rm -rf $PIP_BUILD\n",
     "!pip install psycopg2-binary | tail -n 1\n",
     "!pip install --upgrade watson-machine-learning-client --no-cache | tail -n 1\n",
-    "!pip install ibm-ai-openscale==2.1.1 --no-cache | tail -n 1\n",
+    "!pip install ibm-ai-openscale --no-cache | tail -n 1\n",
```
Comment (markstur) [inline notebooks/OpenScale.ipynb]: lost your version pin here

**Review by markstur — COMMENTED** [review]
prolly OK, but see comments

File: notebooks/OpenScale.ipynb (hunk 1 above)
Comment (scottdangelo) [reply notebooks/OpenScale.ipynb:1444]: I'll change in README

File: notebooks/OpenScale.ipynb (hunk 2 above)
Comment (scottdangelo) [reply notebooks/OpenScale.ipynb]: Fixed

File: notebooks/OpenScale.ipynb (hunk 3 above)
Comment (scottdangelo) [reply notebooks/OpenScale.ipynb]: pinned to new 2.1.7

**Review by rhagarty — APPROVED** [review]

**Pull request merged**